packages/query/src/stacks/balance/account-balance.hooks.ts

```diff
@@ -1,61 +1,17 @@
 import { useQuery } from '@tanstack/react-query';
-import BigNumber from 'bignumber.js';
 
-import type { Money, StxCryptoAssetBalance } from '@leather.io/models';
-import { createMoney, subtractMoney, sumMoney } from '@leather.io/utils';
+import { createMoney } from '@leather.io/utils';
 
 import { useCurrentNetworkState } from '../../leather-query-provider';
-import {
-  AccountBalanceStxKeys,
-  AddressBalanceResponse,
-  accountBalanceStxKeys,
-} from '../hiro-api-types';
 import {
   useMempoolTxsInboundBalance,
   useMempoolTxsOutboundBalance,
 } from '../mempool/mempool.hooks';
 import { useStacksClient } from '../stacks-client';
 import { createGetStacksAccountBalanceQueryOptions } from './account-balance.query';
-
-export function createStxMoney(resp: AddressBalanceResponse) {
-  return Object.fromEntries(
-    accountBalanceStxKeys.map(key => [key, { amount: new BigNumber(resp.stx[key]), symbol: 'STX' }])
-  ) as Record<AccountBalanceStxKeys, Money>;
-}
-
-export function createStxCryptoAssetBalance(
-  stxMoney: Record<AccountBalanceStxKeys, Money>,
-  inboundBalance: Money,
-  outboundBalance: Money
-): StxCryptoAssetBalance {
-  const totalBalance = createMoney(stxMoney.balance.amount, 'STX');
-  const unlockedBalance = subtractMoney(stxMoney.balance, stxMoney.locked);
-
-  return {
-    availableBalance: subtractMoney(totalBalance, outboundBalance),
-    availableUnlockedBalance: subtractMoney(unlockedBalance, outboundBalance),
-    inboundBalance,
-    lockedBalance: createMoney(stxMoney.locked.amount, 'STX'),
-    outboundBalance,
-    pendingBalance: subtractMoney(sumMoney([totalBalance, inboundBalance]), outboundBalance),
-    totalBalance,
-    unlockedBalance,
-  };
-}
-
-function useStxBalanceQuery(address: string) {
-  const client = useStacksClient();
-  const network = useCurrentNetworkState();
-
-  return useQuery({
-    ...createGetStacksAccountBalanceQueryOptions({
-      address,
-      client,
-      network: network.chain.stacks.url,
-    }),
-    select: resp => createStxMoney(resp),
-  });
-}
+import { createStxCryptoAssetBalance } from './create-stx-crypto-asset-balance';
```

Comment:
I just split up this file to help me understand what was going on. I can revert if desired